src/ATE_spyder/ate_spyder/widgets/actions_on/tests/ExcelTestWizard.py

```diff
@@ -0,0 +1,650 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Sep  5 18:56:05 2022
+
+@author: Zlin526F
+
+References:
+    TestWizard
+
+
+todo:  dieser Wizard stellt zur Zeit nur einen einfachen Import von einer Exel liste zur Verfügung.
+       es wird die datenbank und die zugehörigen python files für die einzelnen Test erzeugt.
```

Comment:
overall:

* remove all comments that are not necessary
* better to create github issues for the TODOs and features
* remove commented out code

Reply:
done